id_infrastructure/firestore_uuid_table.py

```diff
@@ -171,11 +171,9 @@ def data_to_uuid(self, data):
 
         exists = uuid_doc_ref.exists
 
-        log.info(f"Ref: {uuid_doc_ref}, exists: {exists}")
```

Comment:
I find this useful as a mental note of no. uids in a project.

Reply:
This isn't telling us how many uids there are though, simply that a uuid for a data item exists.